src/test/rustdoc/mod-title.rs

```diff
@@ -0,0 +1,12 @@
+#![crate_name = "foo"]
```

Comment:
nit: I would put this in the same test file as `item-title` and rename it to `title.rs`. I think they're close enough it makes sense to test them together.